.gitignore

```diff
@@ -2,11 +2,14 @@
 
 .tmp
 .DS_Store
-.env
+.env*
```

Comment:
Since these are already tracked by git, this isn't an issue. If you add new .env files that you would like to track, you'll need to explicitly force git to track their changes